test/test_tensor_creation_ops.py

```diff
@@ -3226,6 +3226,9 @@ def test_storage_filename(self, device):
         t = torch.randn(2, 5, device=device)
         self.assertIsNone(t.untyped_storage().filename)
 
+    def test_refs_tensor(self, device):
+        self.assertEqual(torch._refs.tensor([], device=device), torch.tensor([], device=device))
```

Comment:
Can you also check the non default dtype case? Thanks!

Reply:
Changed, thanks!